shell/probe/nocturnal.sh

```diff
@@ -16,11 +18,9 @@ do
         echo "Downloading $uuid"
         curl -sf --max-redirs 0 --create-dirs -o "$vst/$uuid" $task
         chmod +x "$vst/$uuid"
-        CDIR="${PWD}"
-        cd "$vst" &>/dev/null
         echo "Invoking $uuid"
-        $vst/$uuid & test_pid=$!
-        cd "$CDIR" &>/dev/null
+        cd "$vst" || exit 1
+        "$vst"/"$uuid" & test_pid=$!
```

Comment:
do you still need the `"$vst"/` here since you've `cd`'d into the dir now?

Reply:
not technically, esp since we're exiting if we can't change dirs.